### PR TITLE
Use docker to build service PHP client

### PR DESCRIPTION
**Pull request by kasperg**
Add the swagger-codegen container and link it to the service. We name it
swagger as the container entrypoint is the swagger command.

Add documentation about how to run the command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,14 @@ The project structure is based on [`drupal-composer/drupal-project`](https://git
 	2. Run `docker-compose build`
 	3. Run `docker-compose up`
 3. Profit!
+
+## Client generation
+
+To generate the client code run the following command:
```
Comment (kallehauge) [inline README.md]: Does the reader know the entire context so we can assume that they know what "the client" is?

File: docker/service/Dockerfile [hunk 2]
```diff
@@ -3,7 +3,8 @@ FROM node:5.6
 # The original repo requires the database to run on the same host as the app
 # This fork/branch allows for configuration of database through environment
 # variables. Thus we can run postgres on a separate host using standard images.
-RUN git clone https://github.com/kasperg/dbc-community-service.git -b env_config /usr/src/app
+# Also we need to update the loopback explorer to get Swagger 2.0 specification.
+RUN git clone https://github.com/kasperg/dbc-community-service.git -b biblo-admin /usr/src/app
```
Comment (kallehauge) [inline docker/service/Dockerfile:7]: I've read the PR for dbc.-community-service and saw that they have verbally "accepted" the changes. Have they merged them yet? If yes, then this should be changed :relaxed:

File: README.md (hunk 1 above)
Comment (kasperg) [inline README.md]: I updated the README to clarify.

File: docker/service/Dockerfile (hunk 2 above)
Comment (kasperg) [inline docker/service/Dockerfile:7]: [No - they are not merged yet](https://github.com/DBCDK/dbc-community-service/pulls). We could wait for them to merge before merging this.

Comment (kallehauge) [inline docker/service/Dockerfile:7]: Nah, it's okay. We can adjust it later :+1:

**Comment by kallehauge** [comment]
I'll merge it when Scrutinizer is done running.

**Pull request merged**